README.md

```diff
@@ -15,28 +15,27 @@ reorganization [here](https://www.sylabs.io/2018/03/managing-singularity-branche
 
 # Singularity - Enabling users to have full control of their environment.
 
-Starting a Singularity container "swaps" out the host operating system
-environment for one the user controls!
+Starting a Singularity container "swaps" out the host
+operating system environment for one of the user controls!
```

Comment:
I think the original is more correct here. I anything it could be 'one that the user controls' but not 'one of'